contrib/sarplus/python/pysarplus/SARPlus.py

```diff
@@ -1,13 +1,8 @@
-"""
-This is the one and only (to rule them all) implementation of SAR.
-"""
+"""This is the implementation of SAR."""
```

Comment:
I just realized that we don't have the license here. It's super important to add it here and in all files

Reply:
Added